### ISSUE TITLE
RecordVideo producing black screen

### ISSUE THREAD

**Issue by patchmeifyoucan** [Farama-Foundation/Gymnasium#455]
### Describe the bug

Hello,

I am trying to use the gymnasium RecordVideo wrapper to capture my episodes. While it works perfectly
with other environments, with MuJoCo environments I experience strange behavior. The problem is that
only the first run within an environment instance is recorded correctly. I did not find a way to fix it properly.

One possible way to do it is to recreate both the environment and recorder instance every time I perform
test rollouts, so the issue seems to be related to some internal state which I do not have access to.

Find the code to reproduce the issue below.

### Code example

```shell
import uuid

import gymnasium as gym

env = gym.wrappers.RecordVideo(
    env=gym.make("Humanoid-v4", render_mode="rgb_array"),
    video_folder=f"video/{uuid.uuid4()}",
    episode_trigger=lambda x: True,
    disable_logger=True
)

for i in range(3):
    env.reset()
    done = False
    while not done:
        next_state, reward, terminated, truncated, info = env.step(env.action_space.sample())
        if terminated or truncated:
            break
```

### System info

I am using

Ubuntu 22.04
MuJoCo 2.3.3, gymnasium 0.28.1 and Python 3.10
Thank you very much for your help.

### Additional context

Previously posted in the MuJoCo repo, which was not responsible for it (https://github.com/deepmind/mujoco/issues/824).

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/Gymnasium/issues) in the repo

**Comment by pseudo-rnd-thoughts** [comment]
If you don't use `RecordVideo` and render the frames normally, i.e., `frame = env.render()` and then save the frame (https://stackoverflow.com/questions/902761/saving-a-numpy-array-as-an-image) does this still produce black frames?

**Comment by patchmeifyoucan** [comment]
Hello, thanks for your quick response. If I do it manually, it seems to work. See the code below. 

```python
env = gym.make("Ant-v4", render_mode="rgb_array")
os.makedirs("frames", exist_ok=True)
for epoch in range(5):
    env.reset()
    for step in range(5):
        env.step(env.action_space.sample())
        frame = env.render()
        im = Image.fromarray(frame)
        im.save(f"frames/epoch_{epoch}_step_{step}.jpeg")
```

It also works for other environments. Manually calling `render` with a  `RecordVideo` wrapper also produces valid frames. It fails when automatically producing frames, though.

**Comment by younik** [comment]
Hello, thanks for reporting this!

The problem is with this line: https://github.com/Farama-Foundation/Gymnasium/blob/01308340aec7dd487c714e627d9f248eb646541b/gymnasium/wrappers/monitoring/video_recorder.py#L148

Would you like to open a PR removing it?
This is solved with the new `RecordVideo` in experimental; thus you can use it meanwhile.

**Comment by patchmeifyoucan** [comment]
Hello, thanks for the answer. Using the experimental recorder works for me. 

I can remove the line in the next few days. Are there any guidelines for PRs somewhere?

**Comment by younik** [comment]
> Hello, thanks for the answer. Using the experimental recorder works for me.
> 
> I can remove the line in the next couple of days. Are there any guidelines for PRs somewhere?

Yes, you can refer to this: https://github.com/Farama-Foundation/Gymnasium/blob/main/CONTRIBUTING.md

**Comment by pseudo-rnd-thoughts** [comment]
You should be able to use `gymnasium.experimental.wrappers.RecordVideoV0` which already includes the fix

**Comment by patchmeifyoucan** [comment]
I'm fine with using `RecordVideoV0`, it works. So if the PR is rather unnecessary since it's fixed already in an upcoming version, I'm also happy to avoid going through the process for that one line. 

Anyways, thanks for your help.
